src/card-grid.css

```diff
@@ -0,0 +1,29 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
```

Comment:
What are the @layer components; used for? Is it a special custom class or something else?

Reply:
i simply used it because it was in the example https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply

and i now found the explanation [here](https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply)

> Tailwind will automatically move the CSS within any @layer directive to the same place as the corresponding @tailwind rule, so you don’t have to worry about authoring your CSS in a specific order to avoid specificity issues.
>   
> Any custom CSS added to a layer will only be included in the final build if that CSS is actually used in your HTML, just like all of the classes built in to Tailwind by default.  
> 
> Wrapping any custom CSS with @layer also makes it possible to use modifiers with those rules, like hover: and focus: or responsive modifiers like md: and lg:.

Reply:
Ok, great. Tailwind is a really interesting technology